tools/Microsoft.StandardsPolice/project.json

```diff
@@ -1,6 +1,10 @@
 ﻿{
   "version": "1.0.0-*",
 
+  "compilationOptions": {
```

Comment:
@pakrym I see. @davidfowl is it worth logging a DNX bug to track this? Would this still be an issue in the future?